packages/react-hooks/src/useSetInterval.ts

```diff
@@ -0,0 +1,30 @@
+import { useConst } from './useConst';
+import { useEffect } from 'react';
+
+/**
+ * Returns a self disposing setTimeout function.
```

Comment:
```suggestion
 * Returns a self disposing setInterval function.
```

Reply:
I think I added a similar comment to this function in another PR. @czearing Can you please make a separate PR with just these hooks (integrating the comments I made elsewhere)?